harmony-fe/src/components/post-feed/post-feed.tsx

```diff
@@ -25,7 +25,6 @@ const PostFeed = ({ url, noResultsText = NO_POSTS_HERE }: PostFeedProps) => {
   })
 
   const getPostsIfAvailable = () => {
-    console.log(loading)
```

Comment:
You sure this wont break the site if we remove this?

Reply:
pretty sketchy idk